src/interval_heap.rs

```diff
@@ -399,7 +407,6 @@ mod test {
             let mut ih = IntervalHeap::from_vec(tmp);
             for _ in range(0, 100) {
                 ih.push(rng.next_u32());
-                assert!(is_interval_heap(as_slice(&ih)));
```

Comment:
`collect`'s `Cargo.toml` actually _has_ debugging disabled as of 202076e1. I will change that now that the relevant issue has been fixed in Rust master.